activejob/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+*   Fix ActiveJob log message to correctly report a job failed to enqueue
```

Comment:
```suggestion
*   Fix Active Job log message to correctly report a job failed to enqueue
```

Reply:
Thanks 😓 Fixed in the commit message too.